robot/basestation/static/js/pages/pds.js

```diff
@@ -18,7 +18,7 @@ $(document).ready(() => {
     serviceType: 'ArmRequest' // for now... might change
   })
   // setup a publisher for the pds_command topic
-  let pds_command_publisher = new ROSLIB.Topic({
+  pds_command_publisher = new ROSLIB.Topic({
```

Comment:
this should be removed. It's already being added in `ros-helpers.js`

Reply:
Fixed